components-core/src/main/java/com/adyen/checkout/components/core/internal/data/api/StatusRepository.kt

```
@@ -79,6 +81,9 @@ class DefaultStatusRepository(
             .map {
                 fetchStatus(it)
             }
+            .filterNot {
+                it.exceptionOrNull() is UnknownHostException
```

I would add a comment here to give more context why we are ignoring this exception